src/org/mozartoz/truffle/translator/Translator.java

```diff
@@ -181,6 +185,21 @@ public OzRootNode translateAST(String description, Statement ast, Function<OzNod
 	}
 
 	OzNode translate(StatOrExpr node) {
+		if(node instanceof TailMarkerStatement) {
```

Comment:
missing space after the `if`, normally the Eclipse formatter should take care of this.